Console.Calculator.oksanatk/Program.cs

```diff
@@ -0,0 +1,390 @@
+﻿using Spectre.Console;
+using CalculatorLibrary;
+using Microsoft.CognitiveServices.Speech;
+using System.Text.RegularExpressions;
+
+bool SpeechRecognitionInput = false;
+string? speechKey = Environment.GetEnvironmentVariable("Azure_SpeechSDK_Key");
+string? speechRegion = Environment.GetEnvironmentVariable("Azure_SpeechSDK_Region");
+SpeechConfig speechConfig = SpeechConfig.FromSubscription(speechKey, speechRegion);
+speechConfig.SpeechRecognitionLanguage = "en-US";
+
+string? readResult;
+string[] validOperations = new string[] { "add", "sub", "mult", "div", "sq", "pow", "10x", "sin", "cos", "tan" };
+string[] singleNumberOperations = new string[] { "sq", "10x", "sin", "cos", "tan" };
+Calculator calculator = new Calculator();
+bool endApp = false;
+
+if (args.Contains("--voice-input"))
+{
+    SpeechRecognitionInput = true;
+}
+
+await ShowMainMenu(SpeechRecognitionInput);
+
+async Task ShowMainMenu(bool voiceMode)
```

Comment:
❗ I think with this you have got a `if` `else` nightmare spaghetti. Maybe refactor out into each own specific method for voice / console